<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset resource="root.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar:vertical
{
        border-radius: 2px;
        background: rgb(40, 40, 40);
        width:4px;    
        margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
		background-color: rgb(79, 79, 79);
        min-height: 0px;
		border-radius: 2px;
        width: 3px;
}

QScrollBar::add-line:vertical {
        /*background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0 rgb(32, 47, 130), stop: 0.5 rgb(32, 47, 130),  stop:1 rgb(32, 47, 130));*/
        background-color: #797979;
        border-radius: 2px;
        height: 0px;
        subcontrol-position: bottom;
        subcontrol-origin: margin;
        width: 3px;
}
QScrollBar::sub-line:vertical {
        /*background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop: 0  rgb(32, 47, 130), stop: 0.5 rgb(32, 47, 130),  stop:1 rgb(32, 47, 130));*/
        background-color: #797979;
        height: 0 px;
        border-radius: 2px;
        subcontrol-position: top;
        subcontrol-origin: margin;
        width: 3px;
}

MainWindow {
    background-color: #383838;
}

QToolTip{
    border-radius: 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="left_sidebar">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="sidebar_title">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;</string>
        </property>
        <property name="text">
         <string> Ваши записи</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="searchBar">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>225</width>
            <height>33</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(50, 50, 50);
border-radius: 3px;</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="search_icon">
             <property name="maximumSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">width: 6px;
height: 6px;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="root.qrc">:/search.svg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="search">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>224</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: white;
margin-bottom: 7px;</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="placeholderText">
              <string>Быстрый поиск</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clearSearchBar">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">margin-bottom: 6px;
outline: none;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="root.qrc">
               <normaloff>:/x-lg.svg</normaloff>:/x-lg.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>15</width>
               <height>15</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QScrollArea" name="sidebar">
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">margin: 0;
border-radius: 4px;
background-color: rgb(50, 50, 50);</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="sidebar_content">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>225</width>
           <height>416</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">margin: 0;</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="sidebar_articles">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="tools_panel">
        <item>
         <widget class="QPushButton" name="add_button">
          <property name="maximumSize">
           <size>
            <width>225</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>ArrowCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Создать новую запись</string>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
height: 23px;
background-color: #2e2f30;
border-radius: 3px;
outline: none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="root.qrc">
            <normaloff>:/plus-lg.svg</normaloff>:/plus-lg.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="main_panel">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="title">
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000000</width>
          <height>24</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: white;
text-align: center;</string>
        </property>
        <property name="text">
         <string>Редактор</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="toolset" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">margin-top: 7px;
margin-bottom: 9px;
border-radius: 3px;
background-color: rgb(50, 50, 50);</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="delete_btn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Удалить запись</string>
           </property>
           <property name="styleSheet">
            <string notr="true">outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="root.qrc">
             <normaloff>:/trash.svg</normaloff>:/trash.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="bold">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Жирный текст</string>
           </property>
           <property name="styleSheet">
            <string notr="true">border: none;
margin-left: 5px;
outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="root.qrc">
             <normaloff>:/type-bold.svg</normaloff>:/type-bold.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="underline">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Подчёркивание</string>
           </property>
           <property name="styleSheet">
            <string notr="true">margin: 0;
margin-bottom: 2px;
outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="root.qrc">
             <normaloff>:/type-underline.svg</normaloff>:/type-underline.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>23</width>
             <height>23</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="strikethrough">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Зачёркнутый текст</string>
           </property>
           <property name="styleSheet">
            <string notr="true">outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="root.qrc">
             <normaloff>:/type-strikethrough.svg</normaloff>:/type-strikethrough.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="upload">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Сохранить запись</string>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
           <property name="styleSheet">
            <string notr="true">outline: none;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="root.qrc">
             <normaloff>:/arrow-up-circle.svg</normaloff>:/arrow-up-circle.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>21</width>
             <height>21</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="editor" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>550</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">margin: 0;
border-radius: 5px;
background-color: rgb(50, 50, 50);</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QTextEdit" name="textEdit">
           <property name="styleSheet">
            <string notr="true">color: rgb(238, 238, 236);</string>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="placeholderText">
            <string>Введите вашу запись...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="root.qrc"/>
 </resources>
 <connections/>
</ui>
